tests/common.go

```diff
@@ -77,14 +78,14 @@ func ValidateContext(ctx *scheduler.Context) {
 	ginkgo.Describe(fmt.Sprintf("For validation of %s app", ctx.App.Key), func() {
 		generateSupportBundle(ctx)
 
-		Step(fmt.Sprintf("validate %s app's volumes", ctx.App.Key), func() {
-			ValidateVolumes(ctx)
-		})
-
 		Step(fmt.Sprintf("wait for %s app to start running", ctx.App.Key), func() {
 			err := Inst().S.WaitForRunning(ctx)
 			expect(err).NotTo(haveOccurred())
 		})
+
+		Step(fmt.Sprintf("validate %s app's volumes", ctx.App.Key), func() {
```

Comment:
Any reason to move volume check after app check? The volume check was before applications since volumes are a pre-requisite for a app container to start. So that itself fails, there is no need to `WaitForRunning`.

Reply:
done as per discussed.